src/Rector/Contrib/PHPUnit/SpecificMethodRector.php

```diff
@@ -116,4 +117,20 @@ private function moveArgumentUp(MethodCall $methodCallNode): void
         $funcCall = $methodCallNode->args[0]->value;
         $methodCallNode->args[0] = $funcCall->args[0];
     }
+
+    private function resolveFunctionName(Node $node): ?string
+    {
+        if ($node instanceof FuncCall) {
+            /** @var Name $nameNode */
+            $nameNode = $node->name;
+
+            return $nameNode->toString();
+        }
+
+        if ($node instanceof Empty_) {
+            return 'empty';
+        }
```

Comment:
This is helper function, that can deal with both `FuncCall` and `Empty_`.
Always getting the function name as result.